ci-operator/config/red-hat-data-services/kubeflow/red-hat-data-services-kubeflow-master.yaml

```diff
@@ -34,7 +34,7 @@ releases:
     release:
       architecture: amd64
       channel: fast
-      version: "4.12"
+      version: "4.14"
```

Comment:
It appears that we can leave this at "4.12". I am thinking that doing so may be the right thing to do. It makes sense to test either on the oldest supported on the newest supported version of OCP. And 4.14 is neither of these, whereas 4.12 is oldest supported.